all/test/include.proto

```diff
@@ -2,10 +2,8 @@ syntax = "proto3";
 
 package Messages;
 
-option go_package = "all;all";
+option go_package = "all/test";
```

Comment:
Wondering why the go package name had to be changed?
iirc it's set this way to differentiate and test the source relative option.

Reply:
This is the output of the protoc-gen-go
```
bash ./all/test.sh                       
Testing language go gen/pb-go --with-gateway
protoc-gen-go: invalid Go import path "all" for "all/test/include.proto"

The import path must contain at least one forward slash ('/') character.

See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.

--go_out: protoc-gen-go: Plugin failed with status code 1.
```
Details:
https://github.com/protocolbuffers/protobuf-go/blob/master/compiler/protogen/protogen.go#L259

Reply:
thanks, I realize protoc-gen-go has been upgraded lately.
from my local testing, I think changing the package to `option go_package = "all/;all";` in 
`all/test/include.proto` and `all/test/test.proto` would do.
Can you please try that?

Reply:
Sure. Ill test it out.

Reply:
Works, updated test and proto files.
Pushed.